src/context.cpp

```diff
@@ -87,8 +87,9 @@ namespace Sass {
 
   {
 
-    // add cwd to include paths
-    include_paths.push_back(CWD);
+    // Sass 3.4: The current working directory will no longer be placed onto the Sass load path by default.
```

Comment:
This is actually a bugfix as ruby sass 3.4 removed this behavior and we did not yet ...